mantidproject/mantidimaging
# Add toggle average functionality to ROI selector

Fixes #678 

To test:
- Load data
- Open operations window
- Select ROI
- Right click on the stack
- Select toggle average
- Toggle it back and forth a few times and ensure the ROI works correctly
